tsconfig.json

```diff
@@ -38,8 +38,8 @@
     "strictPropertyInitialization": true,
     "noImplicitThis": true,
     "alwaysStrict": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
```

Comment:
Can we put this back please? I was being lazy

Reply:
done